src/python/grpcio/grpc/_cython/_cygrpc/channelz.pyx.pxi

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018 The gRPC Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cdef class Channelz:
```

Comment:
Why is this a class? All the functions seem stateless. Please just make them module-scoped functions. Emulating Java in Python is not a good idea.